### ISSUE TITLE
open art options

### ISSUE THREAD

**Issue by ThePotatoGuy** [Monika-After-Story/MonikaModDev#3174]
If you're someone who likes to draw and make art, but maybe doesn't want to make big things (like whole outfits/hairstyles), then you can help us make smaller things, like accessories.

We're talking:
* hairpins/hairclips
* rings
* chokers
* necklackes
* other jewlery
* glasses
* face tattoos
* whatever

All we ask is that you draw art following some specific guidelines:

# Sprite requirements
* must be at a resolution of 2560x1700 (twice the size of 1280x850)
* must support all 6 poses. (steepling, arms crossed, restleftpointright, pointright, leaning, down). If an accessory is hidden in a pose, then its okay to not have the sprite for that pose.
* accessory layers should only consist of the accessory, so they can be overlayed over the sprites
* only day versions are needed - we have a special filter for night versions 
* PSD formats preferred, but PNG exports at the mentioned resolution are fine, too.

# Submitting new accessories:
1. Create a new issue on github titled: ACS - <name of your accessory>
2. Link your issue to this one by typing `####` in the body of the issue somewhere.
3. upload or link your PSD/images to the issue.

**Comment by TheRyanReSeT** [comment]
is there a .psd file out there with poses already loaded up? Just curious because it would be easier just to add a layer with the accessory

**Comment by tsmafterlife** [comment]
i have previously helped do a sprite with them which was the disgusted mouth sprite that has now been updated. the problem with just adding a layer over it is that if you put that picture as the same resolution and scale size in the game code it would literally layer that entire image over the pre existing one just for the small accessory or whatever it is your doing. so for the mouth change i made i had to make it the exact same scale size and resolution as well as make sure its placed exactly in that scale to be proportionate with the other mouth only then did it work. granted i could be wrong and making it complex but this is what i learned while working with the sprites with them. then again an accessory may be different then a mouth

**Comment by tsmafterlife** [comment]
@TheRyanReSeT

**Comment by TheRyanReSeT** [comment]
ah thanks for the headsup, do you know where I can get the sprites so i can try work on some accessories? @tsmafterlife

**Comment by tsmafterlife** [comment]
go to the doki doki file you have that runs the monika after story mod, then go to game>mod_assets>monika then you get several files caled (a,c,cg, and f) file cg doesnt show much but inside a, c, and f should be all the compiled sprites organized out. if you have anymore questions or would like to know how the scaling works let me know and if you have discord i can screen share you and go more in depth. let me know how this goes for you though. @TheRyanReSeT

**Comment by tsmafterlife** [comment]
more then likely you will want to see file c and then click def for thats her standard sprites and not her Halloween ones but i listed them all just in case you want to play with other stuff as well. but i need to caution you if you open up one of the sprites from that folder in say photo shop and you make a save to it and not a new copy located in a different folder away from ddlc then you will directly change the official sprite and i have no idea what that will do. so its best to find whatever way works best for you to make an exact replica of the sprite after you find it so that way if you make saved changes it wont affect anything

**Comment by Professor-JMO** [comment]
---

> I needed this.
>> Thanks for the info.

_Bookmarks_

---

**Comment by EntonyEscX** [comment]
Night filter? So I can draw a casual outfit for Monika and send it to you without having to make a night version too?
Curiously, I've been without internet in the past few days (I had to get my dad's internet to download the Christmas update) and yesterday I started making a version of Monika wearing the casual clothes of Satchely's Valentine's Day art to contribute to the mod. Coinsidently, all I only need to do the inclined pose to finish the versions of the day, so I can eviar everything for you when you finish it even though it does not have nightly versions?

**Comment by tsmafterlife** [comment]
this is exactly how it works or that is how it worked for me when i contributed. all you need to do is make sure theirs an art for her in her leaning pose her straight pose and make sure her  arms if she has sleeves all have arts that work with it in those respected poses. no need to do nights as the developers use a night filter. @EntonyEscX would you happen to have photo shop by any chance as it would make the process incredibly easier. if not thats ok it just makes things take longer. but i must point out the art you submit for the mod has to be in the same exact spot as it would be in the mod by correctly scaling it with the original art. if not you will get this (use image as reference) which is what happened to me when i didnt correctly scale the new disgusted sprite correctly and i submitted it.
![monika first attempt full placement failure](https://user-images.githubusercontent.com/43495378/50499613-dbc5e200-09ff-11e9-8df3-9d6b587cdd0a.png)

but if you have it scaled correctly to the image it should come out like it is in the mod  today with the new disgusted sprite.

hope that helps if not im more then willing to answer more questions.

**Comment by EntonyEscX** [comment]
I have no problem with the adjustment. I put my custom ring in my Monika's sprites with every update since before they actually exist in the mod, so I can say that I already have experience with this. 😁

**Comment by EntonyEscX** [comment]
Just one more question. As I did not know about this issue when I was without internet and I started to draw the clothe, I ended up doing everything in the original resolution of 1280x850, okay if I send it like this, or do you want me to increase the dimensions of everything to 2560x1700? I mean, I finished everything and did not want to have to redo everything from scratch in the new resolution, but just selecting the option to increase the size of the image would not make so much difference, since it would only be the inferior quality increasing its area.

**Comment by tsmafterlife** [comment]
ah okay as for the image itself monikas spirit of lower body is in 1280x850 so that would be the resolution you want. just make sure everything is in its right spot and send it here and ill double check you if you want to make sure its how suppose to be. @EntonyEscX

**Comment by ThePotatoGuy** [comment]
Okay just as a heads up, we are very close to having a **base sprite set** that you can use as a guide for outfits/acs/whatever. Once it is ready, it will be linked on this issue.

**Comment by softombre** [comment]
I'm waiting for the base to be released, but I already have a few ideas!

**Comment by ThePotatoGuy** [comment]
just going to note that the only new outfit for next release will be the casual/vday one. Everything else will be looked at right after vday.

**Comment by ThePotatoGuy** [comment]
2019-01-30: PSDs updated. check revision history for changes.

**Comment by ThePotatoGuy** [comment]
2019-02-03: Ribbons added and twilight filter added, check revision history for changes.

**Comment by ThePotatoGuy** [comment]
2019-02-04: Ribbon color settings added and fixes to upright ponytail/bun hairstyles

**Comment by ThePotatoGuy** [comment]
2019-02-07: fixed to color and psd sizing.

**Comment by ThePotatoGuy** [comment]
I've added a spreadsheet of outfits that we would like. Link is in the TODO section.

**Comment by ThePotatoGuy** [comment]
2019-02-19: updated leaning pose, also render order changes from #3782

**Comment by ThePotatoGuy** [comment]
2109-02-24: added upright facial expressions

**Comment by ThePotatoGuy** [comment]
2019-03-09: base psds updated with better looking clavicles from ss

**Comment by ThePotatoGuy** [comment]
2019-05-04: added better hair from @Orcaramelo
